src/demo/components/news/NewsData.ts

```diff
@@ -0,0 +1,28 @@
+import {NewsItem} from './NewsCard';
+
+export function fetchNews(symbol: string) {
+    return new Promise<NewsItem[]>(resolve => {
+        setTimeout(() => {
+            resolve([symbol, symbol, symbol].map(getFakeNews).sort(byDate));
+        }, 100 + Math.random() * 500);
+    });
+}
+
+function getFakeNews(symbol: string): NewsItem {
+    return {
+        date: getRandomDate(),
+        headline: getRandomHeadline(symbol)
+    };
+}
+
+function getRandomDate() {
+    return new Date(new Date().valueOf() - Math.random() * 1e9);
+}
+
+function getRandomHeadline(symbol: string) {
+    return `This is a fake news article for ${symbol}`;
```

Comment:
Suggest go with something like 'Important news about ${symbol}'. We're not explicitly flagging our data as fake elsewhere, and "fake news" is a super politically charged term. Would also remame `getFakeNews` to `getNewsItem` or similar